### PR TITLE
fix(multiple_queries): remove multiple queries one execute

### PR DESCRIPTION
**Pull request by f-bb-toucantoco**
Snowflake doesn't accept multiple queries in a single execution.
## Change Summary

To use specific database and a specific warehouse, we made : 
- 'USE WAREHOUSE my_warehouse; USE DATABASE my_datebase'

The new format is : 
- 'USE WAREHOUSE my_warehouse'
- 'USE DATABASE my_database'

## Related issue number

https://trello.com/c/gPj8WUrq/1238-snowflake-connector-fix-query-fail-because-of-multiple-statement-query

## Checklist

* [X] Unit tests for the changes exist
* [X] Tests pass on CI and coverage remains at 100%

### CODE REVIEW

**Comment by f-bb-toucantoco** [comment]
[[Snowflake connector] fix query fail because of multiple statement query](https://trello.com/c/gPj8WUrq/1238-snowflake-connector-fix-query-fail-because-of-multiple-statement-query)

File: toucan_connectors/snowflake_common.py [hunk 1]
```diff
@@ -177,16 +177,13 @@ def fetch_data(
         limit: Optional[int] = None,
         get_row_count: bool = False,
     ) -> pd.DataFrame:
-        prepare_query: List = []
+        # prepare_query: List = []
```
Comment (raphaelvignes) [inline toucan_connectors/snowflake_common.py:180]: A nettoyer !

**Review by raphaelvignes — APPROVED** [review]
Ptit commentaire à nettoyer :pray: 
Merci pour ce dernier fix ;)

**Comment by f-bb-toucantoco** [comment]
> Ptit commentaire à nettoyer 🙏
> Merci pour ce dernier fix ;)

On verra si c'est le dernier ;)

**Pull request merged**